__test__/pages/forgotpassword.spec.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import configureMockStore from 'redux-mock-store';
```

Comment:
Good job @timi-codes . However I feel you could go ahead and test the whole module with RTL since it can perform majority of the task achieved here by Enzyme

Reply:
Thanks for the feedback. The TTL suggested I leave the test already written with Jest and continue using RTL for other tests.